icons/1panel.svg

```diff
@@ -0,0 +1 @@
8-1.594-5.818-2.9-5.863-2.902-.045-.002-2.706 1.311-5.913 2.917L.29 6.152v5.855l.002 5.855 5.824 2.92c3.203 1.607 5.851 2.923 5.883 2.926.033.002 2.674-1.305 5.87-2.905zm-7.11.92c-.683-.34-1.241-.633-1.242-.65 0-.017 2.312-1.188 5.139-2.603l5.14-2.572V8.127l-3.33-1.666c-1.83-.916-3.328-1.678-3.328-1.694 0-.04 2.446-1.264 2.525-1.264.036 0 1.558.747 3.382 1.66l3.318 1.658-.014 5.183-.014 5.183-5.11 2.56a901.942 901.942 0 0 1-5.168 2.577c-.032.01-.616-.26-1.299-.6zm11.255-4.722c0-.014-.013-.025-.029-.025s-.029.011-.029.025c0 .014.013.025.03.025.015 0 .028-.011.028-.025zm-17.007 1.86-3.312-1.657V6.82L6.81 4.263a1001.94 1001.94 0 0 1 5.17-2.575c.057-.02 2.56 1.211 2.557 1.257-.001.015-2.307 1.179-5.125 2.587L4.29 8.092l-.015 3.9-.015 3.9 3.326 1.665c1.83.916 3.314 1.682 3.3 1.702-.035.05-2.461 1.263-2.52 1.26-.025 0-1.537-.747-3.359-1.659zM4.144 8.035c0-.013-.013-.025-.03-.025-.015 0-.028.012-.028.025 0 .014.013.026.029.026s.029-.012.029-.026zm-.117-.05c0-.014-.013-.025-.029-.025s-.03.01-.03.025c0 .014.014.025.03.025s.03-.011.03-.025zm-.117-.05c0-.015-.013-.026-.029-.026s-.029.011-.029.025c0 .014.013.026.03.026.015 0 .028-.012.028-.026zm-.175-.102c0-.014-.013-.025-.029-.025s-.03.011-.03.025c0 .014.014.025.03.025s.03-.011.03-.025zm-.117-.05c0-.015-.013-.026-.029-.026s-.029.011-.029.025c0 .014.013.026.03.026.015 0 .028-.012.028-.026zm-.116-.051c0-.014-.014-.026-.03-.026s-.029.012-.029.026c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.176-.102c0-.013-.013-.025-.029-.025s-.029.012-.029.025c0 .014.013.026.03.026.015 0 .028-.012.028-.026zm-.116-.05c0-.014-.014-.025-.03-.025s-.029.01-.029.025c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.176-.101c0-.014-.013-.026-.029-.026s-.029.012-.029.026c0 .013.013.025.03.025.015 0 .028-.012.028-.025zm-.116-.051c0-.014-.014-.025-.03-.025s-.029.011-.029.025c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.117-.05c0-.015-.013-.026-.03-.026-.016 0-.029.011-.029.025 0 .014.013.026.03.026.016 0 .029-.012.029-.026zm-.175-.102c0-.014-.014-.025-.03-.025s-.029.011-.029.025c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.117-.05c0-.014-.013-.026-.03-.026-.016 0-.029.012-.029.025 0 .014.013.026.03.026.016 0 .029-.012.029-.026zm-.117-.051c0-.014-.013-.025-.03-.025-.015 0-.028.01-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm-.175-.101c0-.014-.013-.026-.03-.026-.016 0-.029.012-.029.026 0 .013.013.025.03.025.016 0 .029-.012.029-.025zM2.1 7.024c0-.014-.013-.025-.03-.025-.015 0-.028.011-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm-.117-.05c0-.015-.013-.026-.029-.026s-.03.011-.03.025c0 .014.014.026.03.026s.03-.012.03-.026zm-.175-.102c0-.014-.013-.025-.03-.025-.015 0-.028.011-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm9.315 10.226-1.227-.616-.015-3.4-.015-3.4H8.521l.016-.67.016-.67 1.898-.948 1.898-.948.015 2.838c.008 1.56.008 4.096 0 5.633l-.015 2.796zm1.314-5.04V6.433l.277.14c.153.077.324.164.38.194.096.051.102.36.101 5.322v5.268l-.346.165c-.19.09-.36.164-.379.164-.018 0-.033-2.532-.033-5.626z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>1Panel</title><path d="m12 0 10.349 6v12L12 24 1.651 18V6zm0 .326L1.897 6.158v11.664L12 23.653l10.103-5.831V6.158zM8.84 20.523l-5.801-3.349V6.826L12 1.653l2.23 1.287-8.925 5.195v7.73l5.792 3.345zm6.299-17.058 5.822 3.361v10.348L12 22.347l-2.274-1.312 8.969-5.17v-7.73l-5.823-3.362zm-2.137 3.35v2.869l.024 7.666-.691.384-2.18-1.249.008-6.801H8.958L8.95 8.351l3.412-1.965z"/></svg>
```

I've had a play with manually vectorising this based on the PNG in the header [here](https://1panel.cn/docs/). Could @uncenter or another member of the team review, and if happy - can we merge it in?

Reply:
Looks quite good, go ahead and commit it and I'll approve :)

Reply:
For the record, here is how it compares (first with the PNG logo behind the icon, second with the PNG logo in front of the icon):
![Screenshot 2024-06-04 at 15 51 54 (Inkscape)](https://github.com/simple-icons/simple-icons/assets/47499684/b106b852-8f62-492f-a0f6-16dbaba3e0c9)
![Screenshot 2024-06-04 at 15 51 48 (Inkscape)](https://github.com/simple-icons/simple-icons/assets/47499684/f1fb4351-e4d9-4714-a7ac-2842cbb41191)